fdbserver/WaitFailure.actor.cpp

```diff
@@ -26,7 +26,6 @@
 ACTOR Future<Void> waitFailureServer(FutureStream<ReplyPromise<Void>> waitFailure){
 	// when this actor is cancelled, the promises in the queue will send broken_promise
 	state Deque<ReplyPromise<Void>> queue;
-	state int limit = BUGGIFY ? SERVER_KNOBS->BUGGIFY_OUTSTANDING_WAIT_FAILURE_REQUESTS : SERVER_KNOBS->MAX_OUTSTANDING_WAIT_FAILURE_REQUESTS;
```

Comment:
This looks to be the only place that `BUGGIFY_OUTSTANDING_WAIT_FAILURE_REQUESTS` is used. I think the intent was that limit would be used in line 33, but it seems that at some point after this knob was created, someone buggified `MAX_OUTSTANDING_WAIT_FAILURE_REQUESTS` in Knobs.cpp as well, solving the problem in another way. I think we should just delete this buggify knob.

Reply:
Done